.circleci/config.yml

```
@@ -496,6 +501,16 @@ jobs:
           name: Verify locales
           command: yarn verify-locales --quiet
 
+  test-storybook:
+    executor: node-browsers
```

It looks like this job is failing with the error code 137, which often means an out-of-memory error.

I'd recommend using the larger instance size for this job:

```suggestion
    executor: node-browsers-medium-plus
```